### PR TITLE
CI: Update test runners to 8.4 GA

### PR DESCRIPTION
**Pull request by atodorov**
Depends on https://github.com/osbuild/gitlab-ci-terraform/pull/21

### CODE REVIEW

**Review by Gundersanne — APPROVED** [review]
lgtm, adding blocked until that PR is merged.

**Comment by ondrejbudai** [comment]
Note that the new rhel-8.4-ga images are all pre-subscribed by the gitlab executor. Therefore https://github.com/osbuild/image-builder/blob/b8039afe8483f1f6986b3001b09aca56bc8ed125/schutzbot/deploy.sh#L15 can/should be removed.

File: schutzbot/deploy.sh
Comment (Gundersanne) [inline schutzbot/deploy.sh:1]: Needs a whitespace fix.

**Review by Gundersanne — CHANGES_REQUESTED** [review]
Thanks, just one issue.

**Review by Gundersanne — APPROVED** [review]

**Comment by Gundersanne** [comment]
Adding blocked for https://github.com/osbuild/gitlab-ci-terraform/pull/22

**Review by Gundersanne — APPROVED** [review]

**Pull request merged**
